config/prod/xdoan-test-s3.yaml

```diff
@@ -8,19 +8,19 @@ parameters:
   # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
```

Comment:
we should probably just remove this test bucket?  going further i'm wondering if all of these are just for testing HTAN which we already have in production on scicomp.  maybe they can all be junked?

Reply:
I think that's up to @BrunoGrandePhD

Reply:
I realize that this PR has already been merged, but I just wanted to confirm that it will be useful to leave it around while I get up to speed on this project.